### PR TITLE
Pom updates

### PR DESCRIPTION
**Pull request by Steve-Fontes**
Updates to work with Jenkins release process

### CODE REVIEW

**Comment by pilhuhn** [comment]
@Steve-Fontes I think you need to rebase

File: api/pom.xml [hunk 1]
```diff
@@ -1,49 +1,68 @@
...
+        <dependency>
+            <groupId>javax.ws.rs</groupId>
+            <artifactId>javax.ws.rs-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
```
Comment (kwsutter) [inline api/pom.xml]: org.testng is required for development of the api?  or, just for testing?

File: tck/pom.xml [hunk 2]
```diff
@@ -98,6 +93,11 @@
             <artifactId>jackson-jaxrs-json-provider</artifactId>
             <version>2.8.2</version>
         </dependency>
+        <dependency>
+            <groupId>io.opentracing</groupId>
+            <artifactId>opentracing-api</artifactId>
+            <version>0.30.0</version>
```
Comment (kwsutter) [inline tck/pom.xml]: Is this version accurate for opentracing?

File: tck/pom.xml [hunk 3]
```diff
@@ -1,87 +1,82 @@
...
+            <artifactId>geronimo-annotation_1.2_spec</artifactId>
+            <version>1.0</version>
+            <scope>provided</scope>
+            <optional>true</optional>
         </dependency>
+
         <dependency>
-            <groupId>javax.enterprise</groupId>
-            <artifactId>cdi-api</artifactId>
+            <groupId>org.apache.geronimo.specs</groupId>
```
Comment (kwsutter) [inline tck/pom.xml]: Confused by removing the dependency on cdi and replacing it with geronimo atinject...

File: tck/pom.xml (hunk 2 above)
Comment (Steve-Fontes) [reply tck/pom.xml]: This is correct. 0.31.0 is still an RC1, so not using that yet.

File: api/pom.xml (hunk 1 above)
Comment (Steve-Fontes) [reply api/pom.xml]: I'll remove this

File: tck/pom.xml (hunk 3 above)
Comment (Steve-Fontes) [reply tck/pom.xml]: This is from using rest-client as a template. I need to understand why we would use geronimo classes over other implementations.

File: api/pom.xml [hunk 4]
```diff
@@ -1,49 +1,68 @@
...
-
+    <name>MicroProfile Opentracing API</name>
+    <description>Opentracing APIs for MicroProfile :: API</description>
     <dependencies>
         <dependency>
             <groupId>javax.enterprise</groupId>
             <artifactId>cdi-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>javax.inject</groupId>
```
[the remark this answers is not in the document]
Comment (Steve-Fontes) [reply api/pom.xml]: Part of the spec provides for the ability to use "@Inject io.opentracing.Tracer myTracer;". Does this add a dependency between microprofile spec and javax.inject?

**Pull request merged**